pulp_2to3_migration/app/models/repository.py

```diff
@@ -175,7 +175,7 @@ class Pulp2Distributor(BaseModel):
 
     # due to base_path overlap restriction, a distribution can't correspond to multiple pulp 2
     # distributors, thus one-to-one relationship.
-    pulp3_distribution = models.OneToOneField(BaseDistribution,
+    pulp3_distribution = models.OneToOneField(Distribution,
```

Comment:
We will need to migrate/rewrite these columns - but the UUID key itself within should remain the same.

Reply:
...I failed to add the migration to the commit. Sigh. One sec, I'll add and repush.

Reply:
Remember to bump the required version of pulpcore to >=3.12 as well